spec/models/container_scan_spec.rb

```diff
@@ -41,26 +41,26 @@ def get_pod(*_args)
         :image_ref => 'docker://3629a651e6c11d7435937bdf41da11cf87863c03f2587fa788cf5cbfe8a11b9a'
       )
 
-      @image.class.any_instance.stub(:scan_metadata) do |_args|
+      allow_any_instance_of(@image.class).to receive(:scan_metadata) do |instance, _args|
         @job.signal(:data, '<summary><scanmetadata></scanmetadata></summary>')
       end
 
-      @image.class.any_instance.stub(:sync_metadata) do |_args|
+      allow_any_instance_of(@image.class).to receive(:sync_metadata) do |instance, _args|
```

Comment:
Nope, see my other comment, should definitely be _instance and I can do that here.